controller/controller.go

```diff
@@ -94,8 +94,10 @@ func Install(rootDir string, model *model.SystemInstall) error {
 		return err
 	}
 
-	if err = ConfigureNetwork(model); err != nil {
-		return err
+	if !NetworkPassing {
```

Comment:
can you add a comment telling we have the mass installer use case?

Reply:
@dorileo yes, adding now.